README.md

```diff
@@ -33,17 +35,35 @@ import config, { esmConfig } from '@iobroker/eslint-config';
 export default [...config, ...esmConfig];
 ```
 
+## ReactJS project
+
+To use this config in a ReactJS project, your `eslint.config.mjs` would then look like this:
+
+```js
+import config, { reactConfig } from '@iobroker/eslint-config';
+
+export default [...config, ...reactConfig];
+```
+
+It is suggested to create separate `eslint.config.mjs` files for backend and for ReactJS.
+
 ## Changelog
 
 <!--
   Placeholder for the next version (at the beginning of the line):
   ### **WORK IN PROGRESS**
 -->
+
+### **WORK IN PROGRESS**
+
 ### 0.1.4 (2024-09-11)
-* (@GermanBluefox) adjust initial `prettier` rules
+
+-   (bluefox) added ReactJS eslint config file
```

Comment:
Something went wrong here this should be the work in progress, optimaly also use `@GermanBluefox` instead of `bluefox`